src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentPanel.js

```diff
@@ -69,7 +69,7 @@ class DocumentPanel extends Component {
                         <>
                           <div className="default-margin-bottom">
                             Published by{' '}
-                            <b>{doc.metadata.imprint?.publisher}</b>
+                            <b>{doc.metadata.imprint?.publisher} </b>
```

Comment:
this space will be removed in code reformatting by prettier. Please find a better way of creating a space between the components